Minecraft restarts infinitely after session token is remade

error message "Profile not found, please restart your launcher if you recently created your profile":

```
[2024-05-14T15:10:14.567] [DEBUG] Censured - connecting to proxy...
[2024-05-14T15:10:14.786] [DEBUG] Censured - connection to proxy established. forwarding proxied connection to minecraft
[2024-05-14T15:10:15.214] [ERROR] Censured - Minecraft Bot Error:  Error: Profile not found, please restart your launcher if you recently created your profile
    at call (/bridges/censured/node_modules/yggdrasil/src/utils.js:35:40)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async join (/bridges/censured/node_modules/yggdrasil/src/Server.js:20:12)
[2024-05-14T15:10:15.351] [ERROR] Censured - Minecraft bot disconnected from server,attempting reconnect in 5 seconds
```

--- a/src/instance/minecraft/handlers/error-handler.ts
+++ b/src/instance/minecraft/handlers/error-handler.ts
@@ -1,4 +1,5 @@
 import { InstanceType } from '../../../common/application-event.js'
+import { Status } from '../../../common/client-instance.js'
 import EventHandler from '../../../common/event-handler.js'
 import type MinecraftInstance from '../minecraft-instance.js'
 
@@ -25,6 +26,14 @@ export default class StateHandler extends EventHandler<MinecraftInstance> {
         instanceType: InstanceType.MINECRAFT,
         message:
           'Error: does the account own minecraft? changing skin (and deleting cache) and reconnecting might help fix the problem.'
+      })
+    } else if (error.message.includes('Profile not found')) {
+      this.clientInstance.status = Status.FAILED
+      this.clientInstance.app.emit('statusMessage', {
+        localEvent: true,
+        instanceName: this.clientInstance.instanceName,
+        instanceType: InstanceType.MINECRAFT,
+        message: 'Error: Minecraft Profile not found. Deleting cache and reconnecting might help fix the problem.'
       })
     }
   }

